README.md

```diff
@@ -212,37 +116,33 @@ Supported extensions:
 
 ```
-$ darwin push example-dataset -r path/to/images
-Uploading: 100%|########################################################| 3/3 [00:01<00:00,  2.29it/s]
+$ darwin dataset push test /path/to/folder/with/images
+100%|████████████████████████| 2/2 [00:01<00:00,  1.27it/s] 
 ```
 
 Downloads a remote project, images and annotations, in the projects directory 
 (specified in the authentication process [default: `~/.darwin/projects`]).
 
 ```
-$ darwin pull example-project
-Pulling project example-project:latest
-Downloading: 100%|########################################################| 3/3 [00:03<00:00,  4.11it/s]
-```
-
-
-## Table of Arguments
-
-| parser          | parameter                | type               | required  |
-| --------------- | ------------------------ | -----------------  | --------- |
-| `authenticate`  |                          |                    |           |
-| `team`          |                          |                    |           |
-|                 | `team_name`              | str                | False     |
-|                 | `-l`, `--list`           |                    | False     |
-| `create`        | `project_name`           | str                | True      |
-| `local`         |                          |                    |           |
-| `path`          | `project_name`           | str/int            | True      |
-| `pull`          | `project_name`           | str/int            | True      |
-| `remote`        |                          | str                |           |
-| `remove`        | `project_name`           | str                | True      |
-|                 | `-r` `--remote`          | str                | True      |
-| `url`           | `project_name`           | str                |           |
-| `push`          | `project_name`           | str                | True      |
-|                 | `files`                  | str                | True      |
-|                 | `-e`, `--exclude`        | str                |           |
-|                 | `--fps`                  | int                |           |
\ No newline at end of file
+$ darwin dataset pull test 
+Dataset myteam/test:versionname downloaded at /directory/choosen/at/authentication/time.
+```
+
+
+---
+## Usage as a Python library
+
+The framework is designed to be usable as a standalone python library.
+Usage can be inferred from looking at the operations performed in `darwin/cli_functions.py`.
+A minimal example to download a dataset is provided below and a more extensive one can be found in 
+[darwin_demo.py](https://github.com/v7labs/darwin-py/blob/new_README/darwin_demo.py).
+
+```python
+from darwin.client import Client
+from darwin.dataset.identifier import DatasetIdentifier
+
+client = Client.local(team_slug="myteam") 
+dataset_identifier = DatasetIdentifier.from_slug(dataset_slug="test", team_slug="myteam")
+ds = client.get_remote_dataset(dataset_identifier=dataset_identifier)
+ds.pull()    
```

Comment:
```python
client = Client.local() # use the configuration in ~/.darwin/config.yaml
dataset = client.get_remote_dataset(DatasetIdentifier.parse("myteam/test"))
dataset.pull() # downloads annotations and images for the latest exported version
```
I feel like that is nicer minimal introduction

Reply:
adopted